Allow classerloader to be injected

Current method of loading doesn't work on all platforms i.e aws

Request additional method to specify classloader such spring ResourceLoader

Log 
`Caused by: java.nio.file.NoSuchFileException: /BOOT-INF/lib/ortools-natives-linux-7.5.7466.jar!/linux-x86-64/`

I normally fix this using the Spring resource loader

--- a/README.md
+++ b/README.md
@@ -1,11 +1,11 @@
 # ortools-java
 
-Packages [Google OR-Tools](https://github.com/google/or-tools) as a self-extracting jar file at Maven coordinates `com.skaggsm.ortools:ortools-natives-all:7.5.7466`.
+Packages [Google OR-Tools](https://github.com/google/or-tools) as a self-extracting jar file at Maven coordinates `com.skaggsm.ortools:ortools-natives-all:7.5.7467`.
 
 ## How to Use
 
 1. Add my Bintray repository at `https://dl.bintray.com/magneticflux/maven` to resolve the artifacts
-2. Add a dependency on `com.skaggsm.ortools:ortools-natives-all:7.5.7466` in your build tool of choice
+2. Add a dependency on `com.skaggsm.ortools:ortools-natives-all:7.5.7467` in your build tool of choice
 3. Write code:
 ```java
 // Somewhere before using OR-Tools classes

--- a/ortools-loader/src/main/kotlin/com/skaggsm/ortools/OrToolsHelper.kt
+++ b/ortools-loader/src/main/kotlin/com/skaggsm/ortools/OrToolsHelper.kt
@@ -8,15 +8,25 @@ import com.sun.jna.Platform
  * Helper to set up and load OR-Tools.
  */
 object OrToolsHelper {
+
     /**
      * Call before using Google OR-Tools.
      */
     @JvmStatic
-    fun loadLibrary() {
+    @JvmOverloads
+    fun loadLibrary(clazz: Class<*> = this.javaClass) {
+        return loadLibrary(clazz.classLoader)
+    }
+
+    /**
+     * Call before using Google OR-Tools.
+     */
+    @JvmStatic
+    fun loadLibrary(loader: ClassLoader) {
         val path = ClasspathUtils.extractResourcesToTempDirectory(
             "${Platform.RESOURCE_PREFIX}/",
             "ortools-java",
-            this.javaClass
+            loader
         )
         System.load(path.resolve(Platform.RESOURCE_PREFIX).resolve(System.mapLibraryName("jniortools")).toString())
     }

